.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] You have done an excellent job with your project but I would suggest also ignoring the `node_modules` folder for better practices because sometimes by mistake it could be uploaded and it is not necessary (also, it is a really huge folder :sweat_smile:). You could find how to ignore the `node_modules` folder [here](https://www.freecodecamp.org/news/gitignore-what-is-it-and-how-to-add-to-repo/) :wink:

Reply:
Hello,
Thank you very much for your all suggestions.
most of it I forgot to put.
this suggestion added to project